spec/integ/crypto/device-dehydration.spec.ts

```diff
@@ -40,6 +41,29 @@ describe("Device dehydration", () => {
 
         await initializeSecretStorage(matrixClient, "@alice:localhost", "http://test.server");
 
+        const dehydratedDevices = matrixClient.getCrypto()!.dehydratedDevices();
```

Comment:
It might be better to have the crypto object re-emit the dehydration events, rather than exposing the dehydration manager object.

Reply:
+1 to this. Otherwise there is redundancy in the API, between `crypto.dehydratedDevices().start()` and `crypto.startDehydration()`: which one am I supposed to use, as a user of the library?

(You'll also need to have `MatrixClient` re-emit them in `initRustCrypto`, since `CryptoApi` itself is not an event emitter, and give them better names that reflect the fact they are to do with dehydration.)

Basically, do the same thing we've done with `KeyBackupStatus` etc.